core/client/fs/src/main/java/alluxio/client/file/BaseFileSystem.java

```diff
@@ -288,15 +288,27 @@ public List<BlockLocationInfo> getBlockLocations(URIStatus status)
         if (locations.isEmpty() && mFsContext.getPathConf(new AlluxioURI(status.getPath()))
             .getBoolean(PropertyKey.USER_UFS_BLOCK_LOCATION_ALL_FALLBACK_ENABLED)) {
           // Case 2: Fallback to add all workers to locations so some apps (Impala) won't panic.
-          locations.addAll(getHostWorkerMap().values());
-          Collections.shuffle(locations);
+          PropertyKey locKey = PropertyKey.USER_UFS_BLOCK_LOCATION_RETURN_LIMIT;
```

Comment:
```suggestion
          PropertyKey locKey = PropertyKey.USER_UFS_BLOCK_LOCATION_FALLBACK_RETURN_LIMIT;
```

Reply:
let's rename this property key name so it better reflects what it does in the code

Reply:
done